jupyter/d2l-java/chapter_convolutional-modern/googlenet.ipynb

```diff
@@ -0,0 +1,668 @@
+    "\n",
+    "            // Path 1 is a single 1 x 1 convolutional layer\n",
+    "            SequentialBlock p1 = new SequentialBlock().add(\n",
+    "                    Conv2d.builder()\n",
+    "                            .setFilters(c1)\n",
+    "                            .setKernelShape(new Shape(1, 1))\n",
+    "                            .build())\n",
+    "                    .add(Activation::relu);\n",
+    "\n",
+    "            // Path 2 is a 1 x 1 convolutional layer followed by a 3 x 3\n",
+    "            // convolutional layer\n",
+    "            SequentialBlock p2 = new SequentialBlock().add(\n",
+    "                    Conv2d.builder()\n",
+    "                            .setFilters(c2[0])\n",
+    "                            .setKernelShape(new Shape(1, 1))\n",
+    "                            .build())\n",
+    "                    .add(Activation::relu)\n",
+    "                    .add(\n",
+    "                            Conv2d.builder()\n",
+    "                                    .setFilters(c2[1])\n",
+    "                                    .setKernelShape(new Shape(3, 3))\n",
+    "                                    .optPadding(new Shape(1, 1))\n",
+    "                                    .build())\n",
+    "                    .add(Activation::relu);\n",
+    "            \n",
+    "            // Path 3 is a 1 x 1 convolutional layer followed by a 5 x 5\n",
+    "            // convolutional layer\n",
+    "            SequentialBlock p3 = new SequentialBlock().add(\n",
+    "                    Conv2d.builder()\n",
+    "                            .setFilters(c3[0])\n",
+    "                            .setKernelShape(new Shape(1, 1))\n",
+    "                            .build())\n",
+    "                    .add(Activation::relu)\n",
+    "                    .add(\n",
+    "                            Conv2d.builder()\n",
+    "                                    .setFilters(c3[1])\n",
+    "                                    .setKernelShape(new Shape(5, 5))\n",
+    "                                    .optPadding(new Shape(2, 2))\n",
+    "                                    .build())\n",
+    "                    .add(Activation::relu);\n",
+    "\n",
+    "            // Path 4 is a 3 x 3 maximum pooling layer followed by a 1 x 1\n",
+    "            // convolutional layer\n",
+    "            SequentialBlock p4 = new SequentialBlock()\n",
+    "                    .add(Pool.maxPool2dBlock(new Shape(3, 3), new Shape(1, 1), new Shape(1, 1)))\n",
+    "                    .add(Conv2d.builder()\n",
+    "                            .setFilters(c4)\n",
+    "                            .setKernelShape(new Shape(1, 1))\n",
+    "                            .build())\n",
+    "                    .add(Activation::relu);\n",
+    "\n",
+    "            // Concatenate the outputs on the channel dimension\n",
+    "            return new SequentialBlock()\n",
```

Comment:
InceptionBlock itself is a parallelblock, why using SequentialBlock to wrap it?

Reply:
Makes sense. Made the function return a Block type in the declaration and just returning a Parallel Block object.